doc/sdk-cli-reference.md

```diff
@@ -158,30 +162,53 @@ Create app-operator/.gitignore
 
 ## test
 
-### Flags
+### Available Commands
 
-* `-t, --test-location` **(required)** string - location of e2e test files
-* `-k, --kubeconfig` string - location of kubeconfig for kubernetes cluster
-* `-g, --global-init` string - location of global resource manifest yaml file
-* `-n, --namespaced-init` string - location of namespaced resource manifest yaml file
-* `-f, --go-test-flags` string - extra arguments to pass to `go test` (e.g. -f "-v -parallel=2")
-* `-h, --help` - help for test
+#### local - Runs the tests locally
 
-### Use
+##### Flags
+* `--kubeconfig` string - location of kubeconfig for kubernetes cluster
+* `--global-manifest` string - location of global resource manifest yaml file
+* `--namespaced-manifest` string - location of namespaced resource manifest yaml file
+* `--go-test-flags` string - extra arguments to pass to `go test` (e.g. -f "-v -parallel=2")
+* `-h, --help` - help for local
 
-The operator-sdk test command runs go tests built using the Operator SDK's test framework.
+##### Use
 
-### Example:
+The operator-sdk test command runs go tests built using the Operator SDK's test framework.
 
-#### Test
+##### Example:
 
 ```bash
-operator-sdk test --test-location ./test/e2e/
+$ operator-sdk test local ./test/e2e/
 
 # Output:
 ok  	github.com/operator-framework/operator-sdk-samples/memcached-operator/test/e2e	20.410s
 ```
 
+#### cluster - Runs the tests within a cluster
```

Comment:
#### cluster - Runs the e2e tests packaged in an operator image as a pod in the cluster

##### Args

- <image> the operator image that is used to run the tests in a pod